packages/react-theme-provider/package.json

```diff
@@ -36,24 +35,18 @@
     "@types/react-test-renderer": "^16.0.0",
     "@types/webpack-env": "1.16.0",
     "@fluentui/scripts": "^1.0.0",
-    "@fluentui/style-utilities": "^8.0.0-beta.13",
-    "@fluentui/test-utilities": "^8.0.0-beta.5",
     "enzyme": "~3.10.0",
     "enzyme-adapter-react-16": "^1.15.0",
     "react": "16.8.6",
     "react-dom": "16.8.6",
     "react-test-renderer": "^16.3.0"
   },
   "dependencies": {
-    "@fluentui/react-stylesheets": "^1.0.0-beta.5",
-    "@fluentui/theme": "^2.0.0-beta.13",
-    "@fluentui/react-window-provider": "^2.0.0-beta.5",
     "@fluentui/react-hooks": "^8.0.0-beta.10",
+    "@fluentui/react-internal": "^8.0.0-beta.40",
```

Comment:
let's not dep/use from `react-internal` from `react-theme-provider`. we should simply copy all existing impl from root to `/compat`

Reply:
is this why `react-focus` was causing trouble? the circular dependency?

Reply:
React uses shallow equality to compare contexts, do we really want to make them different now?

Reply:
yes, `react-internal` (v8) is meant to use different context as `react-*` (converged). we don't need to support shared context for these 2 worlds